documentation/book/con-metrics-prometheus-resources.adoc

```diff
@@ -15,3 +15,4 @@ When you apply the Prometheus configuration, the following resources are created
 * A `Prometheus` to manage the configuration of the Prometheus pod.
 * A `PrometheusRule` to manage alerting rules for the Prometheus pod.
 * A `Secret` to manage additional Prometheus settings.
+* A `Service` to allow applications running in the cluster to connect to Prometheus (i.e. Grafana using Prometheus as datasource).
```

Comment:
```suggestion
* A `Service` to allow applications running in the cluster to connect to Prometheus (where Grafana is using Prometheus as data source).
```

Reply:
@ppatierno are you ok with this?

Reply:
@PaulRMellor I meant that Grafana is one of the applications that can use Prometheus, reading your suggestion it's not the same.

Reply:
`docu_check` and therefore entire build because string "i.e." is prohibited.

Reply:
@PaulRMellor what for saying "for example" ?